README.md

```diff
@@ -25,13 +25,19 @@ Although the Go SDK does not support most third-party logging solutions natively
 package main
 import (
 	"github.com/sirupsen/logrus"
+	"go.temporal.io/sdk/client"
```

Comment:
I wonder if we should just remove this section (notice there aren't other examples/docs here in the README) and that way people will look at https://github.com/temporalio/samples-go instead (specifically https://github.com/temporalio/samples-go/tree/main/zapadapter).

Thoughts?

Reply:
> and a very austere README may send the wrong message.

The README is already very tiny compared to docs.

> Perhaps some of the sections can be replaced with links to other docs, but keep the section titles?

There aren't really any sections of note besides this one.

I can merge this and we can just table the idea of altering the README more drastically. Sound good?

Reply:
Merging for now is fine.  Or you can just replace the one section we are talking about with a link to the docs for the logger.  I'm good either way.

Reply:
Can merge. Can you update the branch w/ master? and get CI passing?